src/components/App/App.js

```diff
@@ -56,13 +57,20 @@ class App extends Component {
 
     return (
       <div className="app">
-        <Header />
         <main>
-          <Filter
-            onTagSelected={this.handleTagSelect}
-            onCountrySelected={this.handleCountrySelect}
-            onNameSelected={this.handleNameSelect}
-            onToggleFilter={this.toggleFields} />
+          <aside className="sidebar">
```

Comment:
No more header component?

Reply:
Nope I was going to remove it and restructure the UI